examples/docker/02_dind/Dockerfile

```
@@ -0,0 +1,6 @@
+FROM ubuntu:latest
```

```suggestion
FROM ubuntu:noble
```
Noble is a LTS release, and I think it's good to avoid sudden breakage because of sudden change in Ubuntu release.